backend/de.metas.business.rest-api-impl/src/test/java/de/metas/rest_api/v1/ordercandidates/impl/OrderCandidatesRestController_misc_Test.java

```diff
@@ -59,18 +60,18 @@ public static void afterAll()
 	}
 
 	/**
-	 * Asserts that every {@link AttachmentEntry.Type} has a matching {@link JsonAttachmentType} and vice versa
+	 * Asserts that every {@link AttachmentEntryType} has a matching {@link JsonAttachmentType} and vice versa
 	 */
 	@Test
 	void jsonAttachmentTypes()
 	{
 		for (final JsonAttachmentType jsonAttachmentEntryType : JsonAttachmentType.values())
 		{
-			final AttachmentEntry.Type attachmentEntryType = AttachmentEntry.Type.valueOf(jsonAttachmentEntryType.toString());
+			final AttachmentEntryType attachmentEntryType = AttachmentEntryType.valueOf(jsonAttachmentEntryType.name());
```

Comment:
if we need to convert from JSON to AttachmentEntryType in production code (not only tests) i would prefer to have a dedicated method which sits somewhere near the one which converts to JSON.

Reply:
it's just the test AFAIS